benchbuild/utils/wrapping.py

```diff
@@ -77,10 +77,9 @@ def unpickle(pickle_file):
 
 def __create_jinja_env():
     from jinja2 import Environment, PackageLoader
-    return Environment(
-        trim_blocks=True,
-        lstrip_blocks=True,
-        loader=PackageLoader('benchbuild', 'utils/templates'))
+    return Environment(trim_blocks=True,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [By default, jinja2 sets autoescape to False. Consider using autoescape=True or use the select_autoescape function to mitigate XSS vulnerabilities.](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)